internal/services/systemcentervirtualmachinemanager/system_center_virtual_machine_manager_virtual_machine_template_resource.go

```diff
@@ -0,0 +1,209 @@
+	"github.com/hashicorp/go-azure-helpers/lang/response"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/commonschema"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/location"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/extendedlocation/2021-08-15/customlocations"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/systemcentervirtualmachinemanager/2023-10-07/inventoryitems"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/systemcentervirtualmachinemanager/2023-10-07/virtualmachinetemplates"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/systemcentervirtualmachinemanager/2023-10-07/vmmservers"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/sdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/systemcentervirtualmachinemanager/validate"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+type SystemCenterVirtualMachineManagerVirtualMachineTemplateModel struct {
+	Name                                                   string            `tfschema:"name"`
+	Location                                               string            `tfschema:"location"`
+	ResourceGroupName                                      string            `tfschema:"resource_group_name"`
+	CustomLocationId                                       string            `tfschema:"custom_location_id"`
+	SystemCenterVirtualMachineManagerServerInventoryItemId string            `tfschema:"system_center_virtual_machine_manager_server_inventory_item_id"`
+	Tags                                                   map[string]string `tfschema:"tags"`
+}
+
+var _ sdk.Resource = SystemCenterVirtualMachineManagerVirtualMachineTemplateResource{}
+var _ sdk.ResourceWithUpdate = SystemCenterVirtualMachineManagerVirtualMachineTemplateResource{}
+
+type SystemCenterVirtualMachineManagerVirtualMachineTemplateResource struct{}
+
+func (r SystemCenterVirtualMachineManagerVirtualMachineTemplateResource) ModelObject() interface{} {
+	return &SystemCenterVirtualMachineManagerVirtualMachineTemplateModel{}
+}
+
+func (r SystemCenterVirtualMachineManagerVirtualMachineTemplateResource) IDValidationFunc() pluginsdk.SchemaValidateFunc {
+	return virtualmachinetemplates.ValidateVirtualMachineTemplateID
+}
+
+func (r SystemCenterVirtualMachineManagerVirtualMachineTemplateResource) ResourceType() string {
+	return "azurerm_system_center_virtual_machine_manager_virtual_machine_template"
+}
+
+func (r SystemCenterVirtualMachineManagerVirtualMachineTemplateResource) Arguments() map[string]*pluginsdk.Schema {
+	return map[string]*pluginsdk.Schema{
+		"name": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ForceNew:     true,
+			ValidateFunc: validate.SystemCenterVirtualMachineManagerVirtualMachineTemplateName,
+		},
+
+		"location": commonschema.Location(),
+
+		"resource_group_name": commonschema.ResourceGroupName(),
```

Comment:
same as https://github.com/hashicorp/terraform-provider-azurerm/pull/25429/files#r1547192264

Reply:
They can be in different resource group.

Reply:
Updated